sections/collapsible-content.liquid

```diff
@@ -0,0 +1,382 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content layout-{{ section.settings.layout | split: '_' | first }}-container color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content-wrapper{% if section.settings.layout == 'section_container' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
```

Comment:
here we need to add some styling to break the word if it's too long: 
![](https://screenshot.click/21-12-fe4jp-w8eed.png)

Reply:
Might need the `| escape` filter here too